pkg/handlers/internalapi/orders.go

```
@@ -324,8 +324,10 @@ func (h UploadAmendedOrdersHandler) Handle(params ordersop.UploadAmendedOrdersPa
 			file, ok := params.File.(*runtime.File)
 			if !ok {
 				errMsg := "This should always be a runtime.File, something has changed in go-swagger."
+
 				appCtx.Logger().Error(errMsg)
-				return handlers.ResponseForError(appCtx.Logger(), nil), apperror.NewBadDataError(errMsg)
+
+				return ordersop.NewUploadAmendedOrdersInternalServerError(), nil
```

This change was to fix a bug because we were calling `handlers.ResponseForError` incorrectly. I ended up removing the call entirely since we should just return a server error here. We were in effect doing that before, though it was for a different reason (calling the error func incorrectly).